usr/share/rear/layout/save/GNU/Linux/20_partition_layout.sh

```diff
@@ -249,15 +249,20 @@ Log "Saving disk partitions."
             devname=$(get_device_name $disk)
             devsize=$(get_disk_size ${disk#/sys/block/})
 
-            disktype=$(parted -s $devname print | grep -E "Partition Table|Disk label" | cut -d ":" -f "2" | tr -d " ")
+            #Check if blockd is a path of a multipath device.
+            if multipath -c /dev/${blockd} > /dev/null 2>&1 ; then
```

Comment:
@gdha if `multipath` is not installed and setup, then the result of `multipath -c /dev/${blockd}` will not be true; which is good in this case.
So the script will continue by trying to get partition size of `/dev/${blockd}` etc ..

To make it cleaner, we could test if `multipath` command is available before. but it will add extra non necessary `if` condition test.
Tell me what you prefer.